pages/lib/refinery/pages/configuration.rb

```diff
@@ -1,3 +1,5 @@
+require 'set'
```

Comment:
I've forgotten to remove it, thanks for your review.

@anitagraham could you remove it and squash your commits ?